ibtk/include/ibtk/HierarchyMathOps.h

```diff
@@ -840,11 +841,17 @@ class HierarchyMathOps : public SAMRAI::tbox::DescribedClass
      * on each coarse-fine interface are synchronized prior to performing the
      * interpolation.
      *
+     * @note The option @p dst_cf_bdry_synch presently synchronizes values by
```

Comment:
I think this probably should never change ---  with the idea being that the fine value should be "more accurate" than the coarse one, and so we want to make the coarse value agree with the fine value, and then make the overall field continuous by correcting "hanging node" values along the coarse-fine interface.

Reply:
I agree and this should be updated. There's no compelling reason to set fine values from coarse at present, especially if we are only doing this for visualization, so the present version should be fine (albeit I need to finish the "enforce hanging node constraints" code).